test/suggest.lean

```diff
@@ -7,7 +7,7 @@ Authors: Lucas Allen
 import tactic.basic
 
 /- Turn off trace messages so they don't pollute the test build: -/
-set_option trace.silence_suggest true
+-- set_option trace.silence_suggest true
```

Comment:
Is this intentional?

Reply:
Oops, it wasn't, although it should be! The test file now uses the non-interactive tactic for testing, which returns the strings directly rather than tracing them, so there's no need to turn off tracing anymore. I'll remove this line.

Reply:
Or rather, I have already removed this line in #1592, which fixes a bug in `suggest`.

Reply:
```suggestion
set_option trace.silence_suggest true
```